app/views/assessments/index.html.erb

```diff
@@ -165,6 +165,9 @@
                         <span class="new badge blue darken-4" data-badge-caption="unreleased"></span>
                       <% end %>
                     <% end %>
+                    <% if asmt.has_writeup? %>
+                      <span class="new badge orange darken-4" data-badge-caption="writeup" data-url="<%= writeup_course_assessment_path(@course, asmt) %>"></span>
```

Comment:
Would prefer if there was some gap between the writeup flag and the unreleased flag. Now they look too close together.

Reply:
The margin was removed in #2016, but I can add it back